### ISSUE TITLE
Make windows resizable

### ISSUE THREAD

**Issue by bradallred** [gemrb/gemrb#1315]
If we were to implement a simple mirrored tiling scheme for drawing the `View` background, then we could easily make windows resizable further eliminating the need for a widescreen mod. It wouldn't be very difficult for anybody inclined. I dont know if we have an appropriate cursor tho.

**Comment by lynxlynxlynx** [comment]
I imagine that would look good only for the "hud" window backgrounds, as the rest are too complicated. Buttons have bevels, so also wouldn't benefit if scaled this way. Plain scaling of everything would be better in general, but not for the msgwin and actwin, which are the only use cases I can think of.

**Comment by bradallred** [comment]
right, most windows wouldn't work well with suck a simplistic approach. tho most windows dont benefit from resizing either. the portrait window is the main window I would want larger for adding more NPCs. Also the action window so we could add additional controls.

Beyond that we would have to basically have our own backgrounds without the controls. We can add backgrounds to any view via GUIScripting so they could still have one independent of the window.

How does widescreen mod work?

**Comment by lynxlynxlynx** [comment]
Manually resizing chus and preparing the mos files (algo unknown, but it's on github). If we do the 4 hud windows we practically obsolete it.

**Comment by lynxlynxlynx** [comment]
The mod uses weidu's undocumented extend_mos, which from what I can tell only looks at a 100px buffer when cloning. 
https://github.com/WeiDUorg/weidu/blob/a253cea3bc044aee45ad64ca01b7b9df72f62e2f/src/tppatch.ml#L2775
